msfc_ccd/_images/_vectors.py

```diff
@@ -82,4 +82,25 @@ def from_scalar(
         scalar: na.AbstractScalar,
         like: None | na.AbstractExplicitVectorArray = None,
     ) -> Self:  # pragma: nocover
```

Comment:
Oh I see, can you remove this `# pragma: nocover` and add a simple test of `__getitem__`?